assets/index.js

```diff
@@ -10,7 +10,7 @@ const board = {
       title: 'Column Backlog',
       cards: [
         {
-          id: '0206c8d7-4d48-4d97-b867-86fc2d21075d',
```

Comment:
Is there any reason for this line being changed? We've stumbled on a bug with types in `id` (#229) and this line covers that.